cmd/crd-puller/pull-crds.go

```diff
@@ -46,6 +45,7 @@ func main() {
 		Example: "",
 		RunE: func(cmd *cobra.Command, args []string) error {
 			kubeconfigPath := cmd.Flag("kubeconfig").Value.String()
+			resourcesToSync := viper.GetStringSlice("resources-to-sync")
```

Comment:
Mind moving these two variables up outside of `RunE` so we can bind to them?

```go
var (
  kubeconfigPath string
  resources []string
)
```

Reply:
Done.